accelerator/apis.c

```diff
@@ -154,54 +154,51 @@ status_t api_find_transactions_obj_by_tag(
 }
 
 status_t api_receive_mam_message(const iota_client_service_t* const service,
-                                 const char* const obj, char** json_result) {
-  status_t ret = SC_OK;
+                                 const char* const bundle_hash,
```

Comment:
Can you remain the structure of this function? Only `mam_api_add_trusted_channel_pk` needs to be added and maybe some error checks.